pkg/crc/version/version.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (C) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+// The following variables are private fields and should be set when compiling with ldflags, for example --ldflags="-X github.com/code-ready/crc/pkg/version.crcVersion=vX.Y.Z
+var (
+        // The current version of minishift
+        crcVersion = "0.0.0-unset"
+
+        // The SHA-1 of the commit this binary is build off
+        commitSha = "sha-unset"
```

Comment:
👍

Reply:
Unused

Reply:
@gbraad not sure if you have seen in the Makefile
```
 Linker flags
VERSION_VARIABLES := -X $(REPOPATH)/pkg/version.crcVersion=$(CRC_VERSION) \
	-X $(REPOPATH)/pkg/version.commitSha=$(COMMIT_SHA)
```

Reply:
You have seen? https://github.com/code-ready/crc/blob/8badaf3ce8eca458e88488ee4b4946a3cc204d44/cmd/crc/cmd/root.go#L28

Reply:
@gbraad Yes, I guess you are looking for https://github.com/code-ready/crc/issues/32 along with the version package files. Right?